.eslintrc.json

```diff
@@ -3,5 +3,34 @@
     "globals": {
         "jest": "readonly",
         "expect": "readonly"
-    }
-}
+    },
+    "overrides": [
+        {
+            "files": [
+                "*.ts",
+                "*.tsx"
+            ],
+            "extends": [
+                "@exabyte-io/eslint-config",
+                "eslint:recommended",
+                "plugin:@typescript-eslint/eslint-recommended",
+                "plugin:@typescript-eslint/recommended",
+                // "plugin:@typescript-eslint/recommended-requiring-type-checking",
+                "plugin:import/typescript"
+            ],
+            "parser": "@typescript-eslint/parser",
+            "rules": {
+                "@typescript-eslint/ban-ts-comment": "off",
+                "import/no-absolute-path": "off",
+                "import/extensions": "off",
+                "react/forbid-prop-types": "off",
+                "react/jsx-filename-extension": "off",
+                "react/require-default-props": "off",
+                "no-use-before-define": "off",
+                "no-shadow": "off",
+                "@typescript-eslint/no-use-before-define": ["error"]
+            },
+            "plugins": ["@typescript-eslint"]
+        }
+    ]
+}
```

Comment:
https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file